core/circuitbreaker/rule.go

```diff
@@ -114,3 +122,16 @@ func (r *Rule) isEqualsTo(newRule *Rule) bool {
 		return false
 	}
 }
+
+func getStatSlidingWindowBucketCount(r *Rule) uint32 {
```

Comment:
Best can complement a for getStatSlidingWindowBucketCount UT

Reply:
Already have. I've done some renaming. Please have look again.